doc/dev/developer_guide/basic-workflow.rst

```diff
@@ -282,15 +282,15 @@ sub-directory`_ and are run via the `teuthology framework`_.
 .. _`teuthology framework`: https://github.com/ceph/teuthology
 
 The Ceph community has access to the `Sepia lab
-<https://wiki.sepia.ceph.com/doku.php>`_ where :ref:`testing-integration-tests` can be
-run on physical hardware. Other developers may add tags like "needs-qa" to your
+<https://wiki.sepia.ceph.com/doku.php>`_ where `Integration Testing` _ can be
+run on real hardware. Other developers may add tags like "needs-qa" to your
 PR. This allows PRs that need testing to be merged into a single branch and
 tested all at the same time. Since teuthology suites can take hours (even
 days in some cases) to run, this can save a lot of time.
 
 To request access to the Sepia lab, start `here <https://wiki.sepia.ceph.com/doku.php?id=vpnaccess>`_.
 
-Integration testing is discussed in more detail in the :ref:`testing-integration-tests`
+Integration testing is discussed in more detail in the `Integration Testing` _
```

Comment:
ditto

Reply:
there is no link https://ceph--37949.org.readthedocs.build/en/37949/dev/developer_guide/basic-workflow/#integration-tests-aka-ceph-qa-suite